Makefile

```diff
@@ -11,20 +11,24 @@ UTIL=Util/*.cpp
 
 EXECUTABLE=smashbot
 
-all: goals strats tactics chains main util
+all: util goals strats tactics chains main
```

Comment:
For anything that needs `Constants.h` you need to put `util` as a dependency rule.